common/coat_of_arms/template_lists/colored_emblem_lists.txt

```diff
@@ -194,24 +194,27 @@ colored_emblem_texture_lists = {
 			
 			# Warcraft
 			# Trollish
-			100 = "ce_troll_eagle_loa.dds"
-			100 = "ce_troll_mask_zuljin.dds"
-			100 = "ce_draktharon.dds"
-			100 = "ce_sandscalp.dds"
-			100 = "ce_senjin.dds"
-			100 = "ce_troll1.dds"
-			100 = "ce_troll2.dds"
-			100 = "ce_troll3.dds"
-			100 = "ce_troll4.dds"
-			100 = "ce_troll5.dds"
-			100 = "ce_troll6.dds"
-			100 = "ce_troll7.dds"
-			100 = "ce_troll8.dds"
-			100 = "ce_troll9.dds"
-			100 = "ce_troll10.dds"
-			100 = "ce_troll11.dds"
-			100 = "ce_troll12.dds"
-			100 = "ce_troll13.dds"
+			special_selection = {
+				trigger = { scope:culture = { has_graphical_culture = troll_coa_gfx } }
```

Comment:
Add `troll_coa_gfx`, `harpy_coa_gfx` to `coa_african_trigger` so you don't need to have both `troll_coa_gfx` and `central_african_group_coa_gfx` on the culture.

Reply:
I already have `harpy_coa_gfx` and `troll_coa_gfx` in `african_pagan_coa_trigger` ; no `central_african_group_coa_gfx` in either culture .txts. Do you still want me to move them to `coa_african_trigger` instead? There is effectively no difference.